src/main/scala/SurfaceGetter.scala

```diff
@@ -0,0 +1,44 @@
+package shapelens
+
+import cats._, cats.data._
+
+import shapeless._, ops.hlist._
+
+trait SurfaceGetter[P[_], E2, E1]{
+  val apply: Kleisli[P, E1, ?] ~> Kleisli[P, E2, ?]
+}
+
+object SurfaceGetter extends SurfaceGetterImplicits{
+
+  trait Syntax{
+    implicit def readerView[P[_], E2, E1, T](
+        r: Kleisli[P, E1, T])(implicit
+        S: SurfaceGetter[P, E2, E1]): Kleisli[P, E2, T] =
+      S.apply(r)
+  }
+}
+
+trait SurfaceGetterImplicits extends SurfaceGetterLPI{
+
+  implicit def surfaceGetter_singleField[
+      P[_], E2, E1, LE2 <: HList](implicit
+      GE2: Generic.Aux[E2, LE2],
+      S: Selector[LE2, E1]) = new SurfaceGetter[P, E2, E1]{
+    val apply = λ[Kleisli[P, E1,?] ~> Kleisli[P, E2,?]]{
+      _.local(GE2.to _ andThen S.apply)
+    }
```

Comment:
Maybe we should search for something like `MkFieldLens` for `Getter`s in shapeless or even create it (`MkFieldGetter`). I think it's nice because we abstract away the `Generic` ugly stuff, which makes instances hard to read.

Reply:
It doesn't seem to be there a MkFieldGetter, but you are right: there should be one.